py_irt/cli.py

```diff
@@ -36,9 +40,123 @@ def train(
 
 
 @app.command()
-def evaluate():
-    raise NotImplementedError()
+def train_and_evaluate(
+    model_type: str, 
+    data_path: str, 
+    output_dir: str, 
+    epochs: int = 2000, 
+    device: str = "cpu",
+    initializers: Optional[List[str]] = None,
+    evaluation: str = "heldout",
+    seed: int = 42,
+    train_size: float = 0.9,
+):
 
+    console.log(f"model_type: {model_type} data_path: {data_path}")
+    start_time = time.time()
+    config = IrtConfig(model_type=model_type, epochs=epochs, initializers=initializers)
+    if evaluation == "heldout":
+        with open(data_path) as f:
+            items = []
+            for line in f:
+                submission = json.loads(line)
+                model_id = submission["subject_id"]
+                for example_id in submission["responses"].keys():
+                    items.append((model_id, example_id))
+            random.seed(seed)
+            train, test = train_test_split(items, train_size=train_size)
```

Comment:
Should this take in a random seed, or does the call before handle that?

Reply:
I updated this to pass the seed directly to train_test_split.